### DISCUSSION TITLE
error running to_coco.py

### DISCUSSION THREAD

**Discussion by sainisanjay** [bdd100k/bdd100k#48] [Q&A] [answered]
@XiaLiPKU After installing all the dependencies, i get the ImportError: attempted relative import with no known parent package. 
Could you please help me to resolve this issue?
Command used:
`python to_coco.py -i /home/workspace/data/bdd100k_dataset/bdd100k/json/det_v2_val_release.json -o /home/workspace/data/bdd100k_dataset/det_v2_val_release_coco.json --remove-ignore`

OS: Ubuntu 18.04
Python 3.7.5

**Comment by tobiasfshr** [comment, marked as answer]
`python -m bdd100k.label.to_coco -i /home/workspace/data/bdd100k_dataset/bdd100k/json/det_v2_val_release.json -o /home/workspace/data/bdd100k_dataset/det_v2_val_release_coco.json --remove-ignore`
Should work

**Comment by sainisanjay** [comment]
@tobiasfshr  Great! Thanks
